cmd/leanix-k8s-connector/main.go

```diff
@@ -90,7 +90,7 @@ func KubernetesScan(debugLogBuffer *bytes.Buffer) (response *leanix.SelfStartRes
 
 	log.Info("Getting connector config...")
 	log.Info("----------Start----------")
-	log.Infof("LeanIX Kubernetes connector build version: %s", version.VERSION)
+	log.Infof("LeanIX Kubernetes connector build version: %s", version.FULL_VERSION)
```

Comment:
One more thing, maybe we can log both, the version and the full build version?

Reply:
I can do that but eventually both are same with just extra digest right?